File: backend/src/main/java/be/kauffman/kfm/appointment/entity/payload/AppointmentCreatePayload.java
package be.kauffman.kfm.appointment.entity.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class AppointmentCreatePayload {
    Date start_date;
    Date end_date;
    String price;
    String theme;
    String comment;
    String type;
    String category;
}

File: frontend/src/app/module/appointment/model/payload/appointment-create.payload.ts
import {PayloadInterface} from '@shared/model';
import {User} from "@user/model";

export interface AppointmentCreatePayload extends PayloadInterface {
  start_date: Date,
  end_date: Date,
  price: string,
  theme: string,
  comment: string,
  type: string,
  category: string,
  user: User,
  tag: string,
}
